<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="decadesCheckbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Decades</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="yearsCheckbox">
       <property name="text">
        <string>Years</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="quartersCheckbox">
       <property name="text">
        <string>Quarters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="seasonsCheckbox">
       <property name="text">
        <string>Seasons</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="monthsCheckbox">
       <property name="text">
        <string>Months</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="weeksCheckbox">
       <property name="text">
        <string>Weeks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="chartBox">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pandemics</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Wars</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Economic crisis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Weather disaster</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="usHolidaysCheckBox">
       <property name="text">
        <string>US Holidays</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>US President Elections</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Company reports</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Dividends</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="durationComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentText">
          <string>Select Duration</string>
         </property>
         <item>
          <property name="text">
           <string>Select Duration</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 year</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 quarter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 month</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 week</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QGridLayout" name="returnsChartBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
